tests/bsim/bluetooth/audio/prj.conf

```diff
@@ -51,7 +51,7 @@ CONFIG_BT_BAP_BROADCAST_SNK_STREAM_COUNT=2
 CONFIG_BT_ISO_PERIPHERAL=y
 CONFIG_BT_ISO_MAX_CHAN=4
 CONFIG_BT_ISO_TX_MTU=310
-CONFIG_BT_ISO_TX_BUF_COUNT=4
+CONFIG_BT_ISO_TX_BUF_COUNT=36
```

Comment:
Yikes :D I assume this is 2 x `BROADCAST_ENQUEUE_COUNT`?

Reply:
Yes, 2x due to 2 BISes support (though test only uses 1 BIS) and the build error check in the test needs this 2x.